components/TextArea/TextArea.jsx

```diff
@@ -111,6 +161,12 @@ TextArea.defaultProps = {
 };
 
 TextArea.propTypes = {
+  isAutoResize: PropTypes.bool,
+  autoResizeConfig: PropTypes.shape({
+    initialRows: PropTypes.number,
+    minRows: PropTypes.number,
+    maxRows: PropTypes.number,
+  }),
```

Comment:
How about check if `autoResizeConfig` is different of undefined to check if component is auto resize?